rubygem-concurrent-ruby-edge/rubygem-concurrent-ruby-edge.spec

```diff
@@ -16,8 +16,7 @@ Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) >= 1.0.0
-Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) < 1.1.0
+Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) = 1.0.3
```

Comment:
I see it's 1.0.0 on http://mirror.centos.org/centos/7/sclo/x86_64/sclo/sclo-ror42/ - it shouldn't be a problem for dynflow to build if foreman-packaging provides concurrent-ruby 1.0.3, right? It seems to work with mockchain. 

From what I see here, there' s a strict dependency on concurrent-ruby '1.0.3' https://rubygems.org/gems/concurrent-ruby-edge/versions/0.2.3 so 1.0.0 wouldn't work. Are you asking me to submit the updates to `concurrent-ruby` to the SCLo ? Sorry I didn't get what's the issue.